pynecone/config.py

```diff
@@ -129,9 +132,9 @@ class Config(Base):
     username: Optional[str] = None
 
     # The frontend port.
-    port: str = constants.FRONTEND_PORT
+    frontend_port: str = constants.FRONTEND_PORT
```

Comment:
Now I'm thinking maybe we can keep both `port` and `frontend_port` - and throw a deprecation warning on the old one. Just so we don't break anyone's app with the next release.